### PR TITLE
[TEVA-1520] Add jobseeker account feedback feature

### PR DESCRIPTION
**Pull request by cesidio**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1520

## Screenshots of UI changes:
Screenshot 2020-12-16 at 15 36 31
Screenshot 2020-12-16 at 15 35 22
Screenshot 2020-12-16 at 15 35 00
Screenshot 2020-12-16 at 15 37 29

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2525.london.cloudapps.digital>

File: app/components/jobseekers/account_survey_link_component/account_survey_link_component.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  position: sticky;
+  display: block;
+  box-sizing: border-box;
+  bottom: 0;
+  margin-bottom: -40px;
+
+  .account-survey-cta__content {
+    background-color: govuk-colour("orange");
+    padding: govuk-spacing(2);
+    color: govuk-colour('black');
```
Comment (cpjmcquillan) [inline app/components/jobseekers/account_survey_link_component/account_survey_link_component.scss:15]: Single or double quotes? (See above and below)

File: app/views/jobseekers/account_feedbacks/new.html.haml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  .govuk-grid-column-two-thirds
+    = form_for @account_feedback, url: jobseekers_account_feedback_path do |f|
+      = f.hidden_field :back_link
+
+      = f.govuk_error_summary
+
+      %h2.govuk-heading-xl
+        = t('.title')
+
+      = f.govuk_radio_buttons_fieldset(:rating) do
```
Comment (cpjmcquillan) [inline app/views/jobseekers/account_feedbacks/new.html.haml:15]: Can we not use the radio collection?

**Review by cpjmcquillan — APPROVED** [review]
Just a minor couple of comments

**Review by david-mears-dfe — CHANGES_REQUESTED** [review]
Please ask Content Design for a better error message than the default inclusion error.

> Screenshot 2020-12-14 at 22 07 03

**Comment by cesidio** [comment]
> Please ask Content Design for a better error message than the default inclusion error.
> 
> > Screenshot 2020-12-14 at 22 07 03

Yes I sent already 2 messages to ask for that. I am waiting for that content. I am not going to merge it like this

**Review by david-mears-dfe — APPROVED** [review]

**Pull request merged**